incubator/solr/values.yaml

```diff
@@ -22,12 +22,12 @@ terminationGracePeriodSeconds: 180
 # Solr image settings
 image:
   repository: solr
-  tag: 7.6.0
+  tag: 7.7.2
   pullPolicy: IfNotPresent
 
 # Solr pod liveness
 livenessProbe:
-  initialDelaySeconds: 20
+  initialDelaySeconds: 45
```

Comment:
why is this required?

Reply:
Running this on my 16GB mac in minikube the start times were longer.